save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/cvsscalculator/IBaseMetrics.kt

```diff
@@ -0,0 +1,24 @@
+@file:Suppress("FILE_NAME_INCORRECT")
+
+package com.saveourtool.save.cvsscalculator
+
+/**
+ * base interface for all BaseMetrics classes
+ */
+@Suppress("CLASS_NAME_INCORRECT")
+interface IBaseMetrics {
```

Comment:
I think `Temporal Metrics and Environmental Metrics` will be also included to `Cvss` or `CvssValue` \ `CvssVector`?

Reply:
I thought it would be better to implement them as separate date classes

Reply:
In CVSS v4 there will already be 5 different metrics and they will be internally divided into subgroups. It would be too confusing to implement everything in one class

Reply:
But it leads to manipulating with 3 classes instead of a single for user.
I think user should create a single object and set required metrics.

So maybe create a class for `Cvss` which contains `IBaseMetrics`, `Temporal Metrics` and `Environmental Metrics`?

Reply:
A common interface adds a contract for implementing a new version of CVSS: now need to implements 3 different classes and use 3 classes of metrics instead of a singe object

Reply:
Added _Cvss_ class